src/entercommand.js

```diff
@@ -28,14 +35,9 @@ export default class EnterCommand extends Command {
  * Creates a new block in the way that the <kbd>Enter</kbd> key is expected to work.
  *
  * @param {engine.model.Batch} batch A batch to which the deltas will be added.
- * @param {engine.model.Selection} selection
- * @param {Object} options
- * @param {Boolean} options.defaultBlockName The name of the block which should be created when Enter leaves
- * another block. Usually set to `'paragraph'`. For example, when creating a block in `<heading>foo^</heading>`,
- * the result will be `<heading>foo</heading><paragraph>^</paragraph>`.
+ * @param {engine.model.Selection} selection Selection on which the action should be performed.
  */
-export function enterBlock( batch, selection, options = {} ) {
-	const defaultBlockName = options.defaultBlockName;
+function enterBlock( batch, selection ) {
```

Comment:
Why isn't it exposed any more? The command cannot be always reused (undo step integration isn't then possible), so the function can be useful from time to time.

This made me think that perhaps we should allow passing those params (`batch`, `selection`) to all commands which use them. In https://github.com/ckeditor/ckeditor5-enter/pull/28/files#r79292725 I wrote that we need to automate those kind of commands anyway, so perhaps we can go one step further and add also this flexibility.